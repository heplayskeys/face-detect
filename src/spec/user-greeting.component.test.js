import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import UserGreeting from '../components/user-greeting/user-greeting.component';

describe('<UserGreeting />', () => {
	it('returns the correct greeting text and snapshot', () => {
		const name = 'Enzyme';
		const wrapper = shallow(<UserGreeting name={name} />);
		expect(shallowToJson(wrapper)).toMatchSnapshot();
	});
});
