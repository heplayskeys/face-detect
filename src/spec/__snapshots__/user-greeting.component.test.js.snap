// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<UserGreeting /> returns the correct greeting text and snapshot 1`] = `
<div>
  <h1>
    Hello, 
    Enzyme
    !
  </h1>
</div>
`;
